metasrv/tests/it/api/http/metrics.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_base::base::tokio;
+use databend_meta::api::http::v1::metrics::metrics_handler;
+use poem::get;
+use poem::http::Method;
+use poem::http::StatusCode;
+use poem::http::Uri;
+use poem::Endpoint;
+use poem::Request;
+use poem::Route;
+use pretty_assertions::assert_eq;
+
+#[tokio::test]
+async fn test_metrics() -> common_exception::Result<()> {
+    let cluster_router = Route::new().at("/v1/metrics", get(metrics_handler));
+    let response = cluster_router
+        .call(
+            Request::builder()
+                .uri(Uri::from_static("/v1/metrics"))
+                .method(Method::GET)
+                .finish(),
+        )
+        .await
+        .unwrap();
+
+    assert_eq!(response.status(), StatusCode::OK);
```

Comment:
A test has to assert the JSON format.

E.g., in our case, assert expected keys present in the response JSON and the value format(integer, string) is as expected

Reply:
I have a problem about thrift when running tests on my own pc. Can you help me to figure out what's wrong?

```
error[E0432]: unresolved import `thrift::protocol::TSerializable`
  --> path/to/databend/target/debug/build/common-hive-meta-store-2361c9c96a485357/out/hms_patched.rs:16:142
   |
16 | ...ssageType, TInputProtocol, TOutputProtocol, TSerializable, TSetIdentifier, TStructIdentifier, TType};
   |                                                ^^^^^^^^^^^^^ no `TSerializable` in `protocol`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `common-hive-meta-store` due to previous error
```

Reply:
Try to run 'make setup -d'